### ISSUE TITLE
Ars Technica fails to launch with dynamic resources enabled (ModernFix)

### ISSUE THREAD

**Issue by Sethrow2** [zeroregard/Ars-Technica#13]
####  Description
Ars Technica fails to launch when dynamic resources (modernfix mixin) is enabled, this will happen on any version past 1.20.1-1.1.0-a1 including the latest version at this time 1.20.1-1.1.0-b1.
It should be noted that 1.20.1-1.1.0-a6 does have a different error message, but it seems to crash even without ModernFix so I assume it is unrelated.

**To Reproduce**
1. Get the following mods 
    - Ars Nouveau
    - Create
    - Ars Elemental
    - Ars Technica
    - Curios API
    - ModernFix
    - FerriteCore (Not actually needed, it just shuts up ModernFix about a warning, the warning can also be disabled with a config)
2. Enable the dynamic resources mixin (mixin.perf.dynamic_resources=true) in ModernFix's mixin config file (modernfix-mixins.properties), I have also provided the config.
3. Start the game

#### Expected behavior
The game is expected to load onto the main menu but fails to.

#### Crash Log
Here is the crash log with personal details (usernames, system details, etc.) redacted or removed:
https://mclo.gs/gp1diW3

#### Addition Context
[ModernFix Config.zip](https://github.com/user-attachments/files/17457082/ModernFix.Config.zip)

**Comment by zeroregard** [comment]
Thank you very much for opening this ticket with all the details filled!
Can you try out the new beta 2 build I just put up to see if it's working at this point? (it's under review but will likely be available within 5 minutes)

**Comment by Sethrow2** [comment]
Sorry for the late reply! The error has changed, but crashing still occurs (when dynamic resources is enabled). Here is the new crash log:
https://mclo.gs/iSXaTV8

**Comment by zeroregard** [comment]
No worries, that's not a late reply. Thanks for your patience!
I took a look, there's not really any quick fix for having that partial model and not collide with modernfix.

So I have 'fixed' it by simply removing the partial model, that means the Source Motor does not have an arcane shaft when it is powered on, but at least you won't be having this crash anymore. Try out `1.1.0-b3` and check if it's fixed - I tried with the mod and your configuration and it looks good to me.

I might find a way to get this arcane shaft working later on!
